types/minapp-env/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": true,
```

Comment:
You can just set the file contents to `{ "extends": "dtslint/dt.json" }`, rather than explicitly toggling each flag to `true`.

Reply:
(You will need to fix the resulting lint errors either way, though!)

Reply:
done it already.